mne/io/eeglab/eeglab.py

```diff
@@ -80,10 +103,34 @@ def _get_info(eeg, montage, eog=()):
                           for fld in pos_fields)
         get_pos = has_pos and montage is None
         pos_ch_names, ch_names, pos = list(), list(), list()
+        ch_types = list()
         kind = 'user_defined'
         update_ch_names = False
+        err_mesg_shown = False
+        err_ch_set = set()
         for chanloc in eeg.chanlocs:
             ch_names.append(chanloc.labels)
+            if not hasattr(chanloc, 'type'):
+                ch_types.append('eeg')
+            elif not isinstance(chanloc.type, string_types):
+                msg = "'{}' does not use a string variable for its "
+                msg += "channel type. 'eeg' will be assumed, since "
+                msg += "data is sourced from EEGLAB."
```

Comment:
```
msg = ("'{}' does not use a string variable for its "
       "channel type. 'eeg' will be assumed, since "
       "data is sourced from EEGLAB.")
```